packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -331,18 +327,13 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
     );
   }
 
-  Widget _addBackgroundToChild(Widget child) {
-    return DecoratedBox(
-      decoration: BoxDecoration(
-        color: widget.backgroundColor,
-      ),
-      child: child,
-    );
-  }
-
   @override
   Widget build(BuildContext context) {
-    Widget result = DefaultTextStyle(
+    final Color resolvedBackgroundColor = widget.backgroundColor == null
```

Comment:
If this idiom crops up enough, maybe we should allow CupertinoDyanmicColor.resolve(null, context)?

Reply:
https://github.com/flutter/flutter/pull/39927

Reply:
Anything to change now that that PR has been merged?